src/dashboard-below-map-view.js

```diff
@@ -71,7 +73,11 @@ module.exports = cdb.core.View.extend({
   },
 
   _onWidgetsChange: function () {
+    this._widgets.each(function (widget) {
```

Comment:
we can save iterations if we use the `some` iterator returning false after the trigger.

Reply:
Thanks! Btw, I'm still working on this.

Reply:
Oh, I picked up because it's in code review. Sorry about that.

Reply:
Sorry, I have continued the development because this related issue: https://github.com/CartoDB/cartodb/issues/12124